bin/gwpy-wav

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (C) Andrew Lundgren (2017)
+#
+# This file is part of GWpy.
+#
+# GWpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# GWpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GWpy.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""Command line interface to export a sound file
+"""
+import argparse
+
+from gwpy.timeseries import TimeSeries
+from numpy import roll
+from numpy.fft import rfft, irfft
+from scipy.signal import firwin, hann
+from scipy.io import wavfile
+
+__author__ = 'andrew lundgren'
+__email__ = 'andrew.lundgren@ligo.org'
+
+parser = argparse.ArgumentParser(description=__doc__, prog='gwpy-wav')
+parser.add_argument('-c', '--chan', type=str, required=True,
```

Comment:
Wouldn't that make them positional arguments? I've done my best to match the command-line options for gwpy-plot.